rules/library/xcconfig.bzl

```diff
@@ -177,7 +177,7 @@ def _add_copts_from_option(xcspec, name, option, value, value_escaper, xcconfigs
         for arg in new
     ]
 
-def settings_from_xcconfig(xcconfig):
+def copts_from_xcconfig(xcconfig):
```

Comment:
curious about why renaming to copts, since it is not just about copts?

Reply:
I re-named this based on the return value, it's a struct full of copts mostly, I honestly got confused because of the name when I was going through the code for the first time 

But no strong opinions about that, I'm ok with reverting it